src/App.css

```diff
@@ -1,5 +1,112 @@
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,100;0,200;1,100&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Roboto+Slab:wght@300;400;500&display=swap');
+
 * {
   padding: 0;
   margin: 0;
   box-sizing: border-box;
 }
+
+.App {
+  height: 100vh;
+  background-color: #fafafa;
+}
+
+form {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.book-container {
+  width: 90%;
+  margin-top: 90px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  border: 1px solid blue;
+  padding: 10px;
+  background-color: white;
+  margin-bottom: -73px;
+}
+
```

Comment:
- [ ] Kindly make sure your design matches the template(layout, space between items, margin padding...) as the requirement says. 

Your design | the template 
---|---
![image](https://user-images.githubusercontent.com/65068771/179286141-8564ff69-0096-4f17-9d0f-ed154dd642cf.png) | ![image](https://user-images.githubusercontent.com/65068771/179286429-ab2ed629-3428-4112-91f6-765291b93116.png) 
![image](https://user-images.githubusercontent.com/65068771/179286717-ac7c2eee-e8bc-41df-8245-1503bc3c7075.png) | ![image](https://user-images.githubusercontent.com/65068771/179286812-3c52f1c6-6576-49d5-a369-8ea579e47c2f.png)